modules/fakturabank/model/fakturabanksalary.class.php

```diff
@@ -301,7 +301,6 @@ function write($xml) {
         if (isset($_SESSION['oauth_paycheck_sent'])) {
           $data = $_SESSION['oauth_resource']['result'];
           unset($_SESSION['oauth_paycheck_sent']);
-          unset($_SESSION['oauth_resource']);
```

Comment:
Should it be like 
https://github.com/dodoas/lodo/blob/fc249bc3bf17f197ef54c28404bb4594f8865388/modules/fakturabank/model/fakturabank.class.php#L2202-L2206